barcodes_generator_abstract/readme/CONFIGURE.rst

```diff
@@ -4,20 +4,20 @@ To configure this module, you need to:
 * Select a Nomenclature
 * Create or select a rule
 
-.. image:: /barcodes_generator_abstract/static/description/barcode_rule_tree.png
+.. image:: static/description/barcode_rule_tree.png
```

Comment:
```suggestion
.. image:: ../static/description/barcode_rule_tree.png
```

and the rest the same

Reply:
@pedrobaeza thanks for this suggestion. I have updated the paths. Actually I have used that structure from looking at this https://github.com/OCA/partner-contact/blob/14.0/partner_mobile_duplicate_warn/readme/DESCRIPTION.rst
I will take care next time.